### ISSUE TITLE
Animates over Navigation Bar

### ISSUE THREAD

**Issue by patjackson52** [soarcn/BottomSheet#9]
When bottom sheet animates on to the screen it comes over the navigation bar.  Appearing above the system navigation bar would be more desirable.

**Comment by raylee4204** [comment]
@soarcn apologies for late response.

I think if you know the navigation bar is an overlay, there should be an offset from the bottom where the bottom sheet animates in.

**Comment by bsberry** [comment]
@soarcn Just got some cycles to take another crack at this. As noted [here](http://stackoverflow.com/questions/26474125/android-4-4-translucent-status-and-navigation-bars-style-on-android-5-0), setting windowTranslucentNavigation or windowTranslucentStatus on L (haven't tested anywhere else) trumps navigationBarColor/statusBarColor and just sets the color to a medium-opacity black. 

Setting windowTranslucentStatus does seem to have some effect that raises it above the animating bottomsheet, but it then ignores the setting of an opaque black navigationBarColor.

**Comment by bsberry** [comment]
Looking at the native bottomsheet share dialog in L, it looks like Google didn't solve this issue either: their animations appear over the navigation bar too. But they don't actually vertically animate very far; most of the work is done by a 0 -> 1 alpha fade. 

Fixed on our fork by: https://github.com/twotoasters/BottomSheet/pull/3/files . Feel free to close this issue as wontfix.

**Comment by raylee4204** [comment]
@bsberry I checked the animation on google+, and it looks like the enter animation appear over the navigation bar but the exit animation appears below it.

**Comment by soarcn** [comment]
@bsberry Thanks for your correction, setting opaque color for translucentNavigationBar is not possible on L, I just tried that.

I also tested the animation you provided, yes, that would make the "under navigation animation" less likely been seen, and I also seen it in few apps, but they do not have the swiping to dismiss function. Keeping current animation would give user a common experience.

I still consider this is a bug(or feature) on L. Let's see how things proceed ~
